mobile/src/main/java/net/bonysoft/doityourselfie/Extensions.kt

```diff
@@ -55,4 +57,15 @@ fun String.extractName() = this.split("/").last()
 
 const val TOKEN_KEY = "net.bonysoft.doityourselfie.TOKEN"
 
-fun token() = Hawk.get<String>(TOKEN_KEY)
\ No newline at end of file
+fun MainActivity.setStandAloneAuthentication() {
```

Comment:
The PR looks good, but why are you putting this as an extension?
I think this really reduces the visibility, and it doesn't give much compared to having it as a regular method on the class.
I'm curious to hear your thoughts